frontend/awx/administration/instances/InstanceDetails.tsx

```diff
@@ -54,16 +55,27 @@ export function InstanceDetails() {
     useInstanceActions(params.id as string);
   const pageNavigate = usePageNavigate();
   const postRequest = usePostRequest();
+  const activeUser = useAwxActiveUser();
+  const { data } = useGet<Settings>(awxAPI`/settings/system/`);
+
+  const canAddAndEditInstances =
+    (activeUser?.is_superuser || activeUser?.is_system_auditor) && data?.IS_K8S;
   const itemActions: IPageAction<Instance>[] = useMemo(() => {
     const itemActions: IPageAction<Instance>[] = [
       {
         type: PageActionType.Button,
-        selection: PageActionSelection.Single,
+        isHidden: () => data?.IS_K8S === false,
```

Comment:
For instance edit, we will also want to hide the edit button for non execution and hop type instances.